Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -125,6 +125,20 @@ div.bocList table.bocListTable td.bocListTitleCell > span
   display: inline-flex;
   align-items: center;
   padding: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
+  gap: var(--remotion-themed-spacing-small);
+}
+
+div.bocList table.bocListTable th.bocListTitleCell > a > img,
+div.bocList table.bocListTable th.bocListTitleCell > span > img
+{
+  width: var(--remotion-themed-img-width);
+  height: var(--remotion-themed-img-width);
+}
+
+div.bocList table.bocListTable th.bocListTitleCell > span
+{
+  /* Prevent layout changes due to a missing 1px border when switching between sortable/non-sortable. */
```

Comment:
If you want, keep the comment. personally, the variable name in combination with the border makes it clear that we compensate stuff.

If you keep it, I recommend changing the "missing 1px border" to "missing border for anchor".